evap/staff/forms.py

```diff
@@ -227,6 +228,9 @@ def __init__(self, *args, **kwargs):
         if self.instance.pk and not self.instance.can_manager_edit:
             disable_all_fields(self)
 
+        if self.instance.pk:
+            self.instance.old_course = self.instance.course
```

Comment:
hm. i added it during testing because it made creating courses with many evaluations easier. but i can't think of real use cases right now, so we can probably remove everything related to it.

Reply:
should be removed in refactoring